.github/workflows/pyinstaller-windows.yml

```
@@ -0,0 +1,26 @@
+name: Package Application with Pyinstaller
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Package Application
+      uses: JackMcKew/pyinstaller-action-windows@main
+      with:
+        path: mathlibtools
+        spec: leanproject.spec
```

Let's make it clear that these requirements are what are used for pyinstaller
```suggestion
        spec: leanproject.spec
        requirements: pyinstaller-requirements.txt
```
(and rename the requirements file below)